### ISSUE TITLE
TouchScreen working wrong way inside a ScrollContainer

### ISSUE THREAD

**Issue by martinpelli** [godotengine/godot#48133]
**Godot version:**
3.3

**OS/device including version:**
Windows 10 64 bits/Android/iOS

**Issue description:**
If you touch a TouchsScreenButton outside of a ScrollContainer it get touched, with normal buttons this behaviour doesn´t happen.

**Steps to reproduce:**
Download the project and touch/click the screen below the last TouchScrennButton visible, the TouchScreenButton below that is not visible will be pressed.

**Minimal reproduction project:**
https://drive.google.com/drive/folders/1n6g-pTMQqbZcDyp3B_Hw_GKTlLpEcRzt?usp=sharing

**Comment by Calinou** [comment]
TouchScreenButtons aren't meant to be placed inside scrollable areas. For UI purposes, use Button nodes. TouchScreenButtons are reserved for gameplay buttons, such as "jump", "attack" actions and so on. Also, TouchScreenButton inherits from Node2D instead of Control, making them a poor fit for actual UI design.

**Comment by martinpelli** [comment]
@Calinou yeah that what I used, buttons, but I realized that I needed release function that is only available on touchscreen buttons, so I changed buttons to TouchScreenButtons but I faced with this problem. So there is no way to use TouchScreenButtons on Containers?

**Comment by Calinou** [comment]
> So there is no way to use TouchScreenButtons on Containers?

You can make TouchScreenButtons children of a Control node to be able to use anchors on the parent Control node, but it's not particularly convenient and may break under certain circumstances.
